src/components/header.svelte

```diff
@@ -1,19 +1,37 @@
 <script lang="ts">
     import { page } from "$app/stores";
+    import { darkModeNavbar } from "$lib/store";
+    import Burger from "./burger.svelte";
     export let items = [];
     export { classes as class };
 
     let classes: string = "";
+
+    let sideMenuOpen = false;
+
+    const toggleMenu = () => {
+        sideMenuOpen ? closeMenu() : openMenu();
+    };
+    const closeMenu = () => {
+        sideMenuOpen = false;
+    };
+    const openMenu = () => {
+        sideMenuOpen = true;
+    };
 </script>
 
-<nav class="fixed container flex justify-between {classes}">
-    <a href="/">
-        <img src="/assets/logo-TLIP.svg" alt="TLIP logo" />
+<nav
```

Comment:
maybe we could add backdrop-filter: blur(4px) in order that we can see the navbar items properly 
![image](https://user-images.githubusercontent.com/72078237/173128864-72b571a4-d213-449c-83ac-41f763440ec3.png)

Reply:
I also thought about backdrop-filter, I was testing it and I found two problems. One: compatibility [https://caniuse.com/css-backdrop-filter](https://caniuse.com/css-backdrop-filter), second, I thought about adding it to the mobile menu, with the background being transparent but it can only be applied to one element. What should we use?

Reply:
Yes, there are compatibility issues, but you can fix them as we did in previous projects :wink: